azure-devops/cross-build-test.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
Now that I'm seeing what this is shaping up to be I think it might not make sense to have this file at all. Just create an additional stage in the `azure-pipelines.yml` and have it include the `cross-test.yml`.